src/einsteinpy/tests/test_plotting/test_scattergeodesicplotter.py

```diff
@@ -8,7 +8,10 @@
 
 
 @pytest.fixture()
-def dummy_data():
+def create_test_datas():
+    """
```

Comment:
Why docstring here?

Reply:
I thought I had to document the test functions I made?